domain/domain.go

```diff
@@ -888,6 +888,8 @@ func (do *Domain) UpdateTableStatsLoop(ctx sessionctx.Context) error {
 	statsHandle := handle.NewHandle(ctx, do.statsLease)
 	atomic.StorePointer(&do.statsHandle, unsafe.Pointer(statsHandle))
 	do.ddl.RegisterEventCh(statsHandle.DDLEventCh())
+	do.wg.Add(1)
+	go do.loadStatsWorker()
```

Comment:
Can we move this out of UpdateTableStatsLoop?

Reply:
Why?

Reply:
never mind it doesn't matter much